diff --git a/hpm_sdk/components/enet_phy/rtl8211/hpm_rtl8211.c b/hpm_sdk/components/enet_phy/rtl8211/hpm_rtl8211.c
index dd5303e6a..85a951b35 100644
--- a/hpm_sdk/components/enet_phy/rtl8211/hpm_rtl8211.c
+++ b/hpm_sdk/components/enet_phy/rtl8211/hpm_rtl8211.c
@@ -79,13 +79,13 @@ bool rtl8211_basic_mode_init(ENET_Type *ptr, rtl8211_config_t *config)
         data |= RTL8211_BMCR_DUPLEX_SET(config->duplex);                                                /* Set duplex mode */
     }
 
+    enet_write_phy(ptr, RTL8211_ADDR, RTL8211_BMCR, data);
+
     /* check the id of rtl8211 */
     if (rtl8211_check_id(ptr) == false) {
         return false;
     }
 
-    enet_write_phy(ptr, RTL8211_ADDR, RTL8211_BMCR, data);
-
     return true;
 }
 
